apps/api/src/modules/auth/auth.controller.ts

```diff
@@ -49,4 +49,10 @@ export class AuthController {
   login(@Body() loginDto: LoginDto, @Res({ passthrough: true }) res: Response) {
     return this.authService.verfyPassword(loginDto, res)
   }
+
+  @Get('logout')
```

Comment:
I think logout should be `POST` instead
Ref: https://stackoverflow.com/a/31545894